dev/com.ibm.ws.jdbc/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -768,6 +768,8 @@ portNumber=Port number
 portNumber.desc=Port on which to obtain database connections.
 prefetch=Prefetch
 prefetch.desc=JDBC driver property: prefetch.
+preparedStatementCacheQueries=Prepared Statement Cache Queries
+preparedStatementCacheQueries.desc=JDBC driver property: preparedStatementCacheQueries.
```

Comment:
Change
"JDBC driver property: preparedStatementCacheQueries."
to
"The JDBC driver property is preparedStatementCacheQueries."

Reply:
I don't agree with this change unless you are willing to change every other occurrence in this file.  This file currently has 431 messages using the form, "JDBC driver property: ...", and 0 occurrences using the recommended form, "The JDBC driver property is ...".   Please ensure that all of them remain consistent.

Reply:
@njr-11 I understand, and thanks for this information. Disregard my previous suggestion on this one.